### ISSUE TITLE
How is VSAIT (ours) derived in the table?

### ISSUE THREAD

**Issue by xiyanbupapang** [facebookresearch/vsait#1]
Thank you for your perfect work! I have run your code successfully as per your instructions, but still have a small question for you. In section 4.5 of the paper there are two tables showing the excellence of VSAIT, how is VSAIT (ours) measured in the table? If you can, could you please give me all the corresponding assessment code, I look forward to your answer!

**Comment by jdtheiss** [comment]
Thanks for your interest in our work. For all Table 1 evaluation, we followed the same methods described in Jia et al. (2021; https://arxiv.org/pdf/2012.04932.pdf). You can also find more information in our supplemental: https://arxiv.org/pdf/2209.02686.pdf.

Below are the details for each evaluation:
For our GTA->Cityscapes evaluation, we followed the method described in the supplemental of Jia et al. (2021) by using the pre-trained checkpoint "mobilenetv3_small_cityscapes_trainfine" (downloaded from here: https://github.com/tensorflow/models/blob/master/research/deeplab/g3doc/model_zoo.md) and evaluation code similar to that found here: https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/tree/master/scripts/eval_cityscapes.

For the Map->Photo/Photo->Map evaluation (pixel accuracy), we likewise followed Jia et al. (2021) and evaluation code similar to that found here: https://github.com/hufu6371/GcGAN/blob/master/evaluation/evaluate_maps.py.

For Table 2 KID evaluation, we followed Richter et al. (2021; https://arxiv.org/pdf/2105.04619.pdf) and evaluation code similar to that found here: https://github.com/abdulfatir/gan-metrics-pytorch/blob/master/kid_score.py.

**Comment by xiyanbupapang** [comment]
> 

Your guidance was very helpful and I thank you very much for such a prompt and detailed reply!

**Comment by jdtheiss** [comment]
Glad I could help!

**Comment by shahaf1313** [comment]
Hey again :) 
It seems that you have used tensorflow model of deeplab. How did you manage to load a tensorflow checkpoint onto a pytorch framework? 
Thanks,
Shahaf

**Comment by jdtheiss** [comment]
> Hey again :) It seems that you have used tensorflow model of deeplab. How did you manage to load a tensorflow checkpoint onto a pytorch framework? Thanks, Shahaf

Hi Shahaf,

We didn't load it into pytorch; we just converted the checkpoint to onnx for the semantic segmentation evaluation.

-Justin

**Comment by shahaf1313** [comment]
Do you have the conversion code? Can you please share it?
I have never delt with onnx and I don't know what should be done to convert this checkpoint...

**Comment by jdtheiss** [comment]
I do not, but it should be fairly straightforward if you follow online tutorials.

**Comment by shahaf1313** [comment]
Online tutorials of ONNX?

**Comment by jdtheiss** [comment]
Yes, e.g., https://github.com/onnx/tutorials#converting-to-onnx-format.

**Comment by shahaf1313** [comment]
Thanks. 
One more question regarding the KID - how many images of Cityscapes did you use as input? And how many translated images of GTA5?

**Comment by jdtheiss** [comment]
In all cases, we used the held-out validation data from each dataset for evaluation. You can find the counts for those and the other datasets in the paper and/or supplemental.
